### PR TITLE
[tests-only][full-ci] added test to update a file with empty content

### PR DESCRIPTION
**Pull request by PrajwolAmatya**
## Description
In this PR, added test to update file with empty content.
Added scenarios:
```feature
Scenario Outline: user updates a file with empty content
Scenario Outline: user updates a file inside a folder with empty content
Scenario Outline: user updates a shared file with empty content
Scenario: user updates a file inside a project space with empty content
Scenario: user updates a file inside a shared space with empty content
```

## Related Issue

- https://github.com/owncloud/ocis/issues/8044

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/coreApiWebdavUpload1/uploadFile.feature [hunk 1]
```diff
@@ -366,3 +366,46 @@ Feature: upload file
...
+
+
+  Scenario Outline: user updates a shared file with empty content
+    Given using <dav-path-version> DAV path
+    And user "Brian" has been created with default attributes and without skeleton files
+    And user "Alice" has uploaded file with content "file with content" to "/textfile.txt"
+    And user "Alice" has shared file "/textfile.txt" with user "Brian" with permissions "read,update"
+    When user "Brian" uploads file with content "" to shared resource "Shares/textfile.txt" using the WebDAV API
+    Then the HTTP status code should be "204"
+    And the content of shared resource "Shares/textfile.txt" for user "Brian" should be ""
```
Comment (saw-jan) [inline tests/acceptance/features/coreApiWebdavUpload1/uploadFile.feature:405]: there should be step available.

File: tests/acceptance/features/coreApiWebdavUpload1/uploadFile.feature [hunk 2]
```diff
@@ -366,3 +366,46 @@ Feature: upload file
...
+    Then the HTTP status code should be "204"
+    And the content of file "testFolder/textfile.txt" for user "Alice" should be ""
+    Examples:
+      | dav-path-version |
+      | old              |
+      | new              |
+      | spaces           |
+
+
+  Scenario Outline: user updates a shared file with empty content
```
Comment (saw-jan) [inline tests/acceptance/features/coreApiWebdavUpload1/uploadFile.feature:398]: also, we can add for project space scenario in other feature file

File: tests/acceptance/features/coreApiWebdavUpload1/uploadFile.feature [hunk 3]
```diff
@@ -366,3 +366,71 @@ Feature: upload file
...
+    Then the HTTP status code should be "204"
+    And the content of file "/textfile.txt" for user "Alice" should be ""
+    Examples:
+      | dav-path-version |
+      | old              |
+      | new              |
+      | spaces           |
+
+
+  Scenario Outline: user updates a file with empty content in a folder
```
Comment (saw-jan) [inline tests/acceptance/features/coreApiWebdavUpload1/uploadFile.feature:384]:
```suggestion
  Scenario Outline: user updates a file with empty content in a folder
```
```suggestion
  Scenario Outline: user updates a file inside a folder with empty content
```

File: tests/acceptance/features/coreApiWebdavUpload1/uploadFile.feature [hunk 4]
```diff
@@ -366,3 +366,71 @@ Feature: upload file
...
+    And for user "Brian" the content of the file "/test.txt" of the space "Shares" should be ""
+    And the content of file "/textfile.txt" for user "Alice" should be ""
+    Examples:
+      | dav-path-version |
+      | old              |
+      | new              |
+      | spaces           |
+
+
+  Scenario: user updates a file with empty content inside project space
```
Comment (saw-jan) [inline tests/acceptance/features/coreApiWebdavUpload1/uploadFile.feature:414]:
```suggestion
  Scenario: user updates a file inside a project space with empty content
```

File: tests/acceptance/features/coreApiWebdavUpload1/uploadFile.feature [hunk 5]
```diff
@@ -366,3 +366,71 @@ Feature: upload file
...
+    Given using spaces DAV path
+    And the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
+    And user "Alice" has created a space "new-space" with the default quota using the Graph API
+    And user "Alice" has uploaded a file inside space "new-space" with content "file with content" to "textfile.txt"
+    When user "Alice" uploads a file inside space "new-space" with content "" to "textfile.txt" using the WebDAV API
+    Then the HTTP status code should be "204"
+    And for user "Alice" the content of the file "/textfile.txt" of the space "new-space" should be ""
+
+
+  Scenario: user updates a file with empty content inside a shared space
```
Comment (saw-jan) [inline tests/acceptance/features/coreApiWebdavUpload1/uploadFile.feature:424]:
```suggestion
  Scenario: user updates a file inside a shared space with empty content
```

**Review by saw-jan — COMMENTED** [review]
others look good

File: tests/acceptance/features/coreApiWebdavUpload1/uploadFile.feature [hunk 6]
```diff
@@ -366,3 +366,71 @@ Feature: upload file
...
+    And for user "Brian" the content of the file "/test.txt" of the space "Shares" should be ""
+    And the content of file "/textfile.txt" for user "Alice" should be ""
+    Examples:
+      | dav-path-version |
+      | old              |
+      | new              |
+      | spaces           |
+
+
+  Scenario: user updates a file inside a project space with empty content
```
Comment (saw-jan) [inline tests/acceptance/features/coreApiWebdavUpload1/uploadFile.feature:414]: Could you check if these scenarios work on reva-edge or not?

**Review by grgprarup — COMMENTED** [review]
LGTM :+1:

**Review by saw-jan — APPROVED** [review]
LGTM :+1: :sparkles:

**Review by SagarGi — APPROVED** [review]
LGTM :+1:

**Pull request merged**